commit 1503088abd917f40368d5d80ce1c1e2c83ea5a96 (refs/changes/07/1407/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-02T14:41:23-08:00 (2 years, 8 months ago)
    
    joyent/node-triton#166 see fix for joyent/node-sshpk#22 before release
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 6adc554..1ce953d 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -7,6 +7,10 @@ Known issues:
 
 ## not yet released
 
+- [joyent/node-triton#166] Update sshpk to fix issue with the TLS client cert
+  created by `triton profile docker-setup` so that it doesn't create a cert
+  that Go's TLS library doesn't like.
+
 - [joyent/node-triton#156] Providing all required profile options as
   command line flags (account, url, keyId) no longer produces an
   incomplete profile error.
diff --git a/package.json b/package.json
index c96f85b..4bfe433 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "rimraf": "2.4.4",
     "semver": "5.1.0",
     "smartdc-auth": "2.5.2",
-    "sshpk": "1.10.1",
+    "sshpk": "1.10.2",
     "sshpk-agent": "1.4.2",
     "strsplit": "1.0.0",
     "tabula": "1.9.0",
